builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -340,7 +340,7 @@ function store.get_formspec(dlgdata)
 
 	local W = 15.75
 	local H = 9.5
-
+    
```

Comment:
whitespace

Reply:
Do you suggest to add a whitespace in the empty line? What for?

Reply:
the suggestion is to *remove* whitespace from that line :p

Reply:
Done.